apps/ecommerce/frontend/src/components/Config/ConfigPage.tsx

```diff
@@ -111,26 +112,27 @@ const ConfigPage = () => {
     setParameters({ ...parameters, [key]: value });
   };
 
-  const loadingSkeleton = () => {
+  const LoadingSkeleton = () => {
     return (
       <Skeleton.Container>
-        <Skeleton.BodyText numberOfLines={4} />
+        <Skeleton.DisplayText />
+        <Skeleton.BodyText offsetTop={40} numberOfLines={4} />
       </Skeleton.Container>
     );
   };
 
   return (
     <>
-      <Box className={styles.background} testId="configPageContainer" />
+      <Box className={styles.background(providerConfig.color ?? '')} testId="configPageContainer" />
```

Comment:
There's one smell I have here which is related to checking isLoading twice.

This container shouldn't be just some element at the top which you extend and manipulate the css to create the background. It should contain all of its children subcomponents.

More simply, this Box component should wrap everything below here. You should base your height off that and use a minHeight in the css. Once you have this element as an actual container, you can then just do your skeleton loading check once for both the logo and the field contents